skimage/exposure/tests/test_exposure.py

```diff
@@ -237,10 +237,16 @@ def test_rescale_in_range_clip():
 
 
 def test_rescale_out_range():
+    """Check that output range is correct.
+
+    .. versionchanged:: 0.17
+        This function used to return dtype matching the input dtype. It now
+        matches the output.
+    """
     image = np.array([-10, 0, 10], dtype=np.int8)
     out = exposure.rescale_intensity(image, out_range=(0, 127))
-    assert out.dtype == np.int8
-    assert_array_almost_equal(out, [0, 63, 127])
+    assert out.dtype == np.float_
```

Comment:
```suggestion
    assert out.dtype == np.float64
```
I find this more explicit or is there a reason you prefer `float_`? Same in the test below...

Reply:
I prefer `float_` because it is the default float value for that platform. In some platforms, `np.zeros(a_shape)` returns float32, not float64. I think we should return this default type, and that is what `float_` evaluates to.

Reply:
Ah that makes sense. :+1:  Strange that my search pointed me to an old NumPy doc version with the statement "[Shorthand for float64](https://docs.scipy.org/doc/numpy-1.13.0/user/basics.types.html#array-types-and-conversions-between-types)" first.